cmd/execution_config.go

```diff
@@ -63,6 +64,7 @@ func (exeConf *ExecutionConfig) SetupFlags(flags *pflag.FlagSet) {
 	flags.Uint32Var(&exeConf.mTrieCacheSize, "mtrie-cache-size", 500, "cache size for MTrie")
 	flags.UintVar(&exeConf.checkpointDistance, "checkpoint-distance", 20, "number of WAL segments between checkpoints")
 	flags.UintVar(&exeConf.checkpointsToKeep, "checkpoints-to-keep", 5, "number of recent checkpoints to keep (0 to keep all)")
+	flags.BoolVar(&exeConf.outputCheckpointV5, "outputCheckpointV5", false, "output checkpoint file in v5")
```

Comment:
@fxamacker added a bool flag for using v5

Reply:
>Instead of using bool flag to indicate v5, what do you think about using int flag for checkpoint version and set default to MaxVersion like this:

I've set the default to use MaxVersion. But I think it still makes sense to use bool flag here. 

We don't want to support old versions, but only v5. 

Using bool can reduce human mistake, and we will anyway remove this flag after next spork.